src/test/resources/Features/api-test/announcementController.feature

```diff
@@ -0,0 +1,8 @@
+@Regression
+Feature: Verify announcement-controller endpoints return correct status code
+
+Scenario Outline: Verify announcement-controller endpoints return correct status codes for a non-logged in user
+    Then the "<API>" endpoint returns correct "<Status Code>"
+    Examples:
+      |API                                 |Status Code|
+      |/announcements                      |200        |
```

Comment:
This is the only one that doesn't have a newline at the end of the file

Reply:
Done!